source/tutorial/autocomplete-component.md

```diff
@@ -0,0 +1,202 @@
+As they search for a rental, a user might also want to narrow their search
+ to a specific city. Let's build a component that will let them search for
+  properties within a city, and also suggest cities to them as they type.
+
+To begin, let's generate our new component. We'll call this component 
+`filter-listing`.
+
+```shell
+ember g component filter-listing
+```
+As before, this creates a Handlebars template 
+(`app\templates\components\filter-listing.hbs`) and a JavaScript file 
+(`app\components\filter-listing.js`).
+
+The Handlebars template looks like this:
+
+```app\templates\components\filter-listing.hbs
```

Comment:
Our paths run the other direction in all our other examples, can you switch these?